libs/utils/env.py

```diff
@@ -439,6 +441,54 @@ def _init_target(self, force = False):
             self.__modules = ['bl', 'cpufreq']
             platform = Platform(model='pixel')
 
+        # Initialize gem5 platform
+        elif self.conf['board'].upper() == 'GEM5':
+            self.nrg_model = None
+            self.__modules=['cpufreq']
+
+            system_info = self.conf['board_settings']['system']
+            simulator_info = self.conf['board_settings']['simulator']
+            simulator_args = ""
+            platform_args = ""
+
+            # Get gem5 binary arguments
+            for args in simulator_info['args']:
+                simulator_args += args + " "
+            # Required for telnet terminal
+            simulator_args += "--listener-mode=on"
+
+            # Get platform arguments
+            for args in system_info['platform']['args']:
+                platform_args += args + " "
+            # Mandatory arguments
+            platform_args += " --kernel "     + system_info['kernel'] + \
+                             " --dtb "        + system_info['dtb'] + \
```

Comment:
Nit: use implicit continuation instead of `\` (i.e. wrap expressions in brackets then you don't need to escape the newline)